### PR TITLE
Add python-openbabel for Trusty

### PR DESCRIPTION
**Pull request by s-preston**
This is a package addition request from Alex P.

### CODE REVIEW

**Comment by kaduk** [comment]
+1
(We already have regular openbabel)

**Comment by andersk** [comment]
+1

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+debathena-thirdparty (1.2.11.2) unstable; urgency=low
+
+  * Add python-openbabel for Trusty
```
Comment (andersk) [inline debian/changelog]: Our Git hook rejects trailing whitespace on this line.

Comment (s-preston) [inline debian/changelog]: Fixed. Thank you.

**Comment by andersk** [comment]
The hook examines every commit individually, so it is not satisfied by tacking a fixup commit onto the end. You need to squash the fixup into the commit it fixes (with `git commit --amend` before you made the fixup into its own commit, or `git rebase -i` afterwards). This is a good idea in general because it results in readable history that interacts correctly with tools like `git blame` and `git bisect`.

I’ve fixed it for you, in this case.

**Pull request merged**